CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 
 * [#392](https://github.com/mozilla-rally/rally-core-addon/pull/392): Implement the new consent logic to prevent running side-loaded studies.
 
+* [#383](https://github.com/mozilla-rally/rally-core-addon/pull/383): Adds IRB consent notice into the Core Add-On.
```

Comment:
```suggestion
* [#383](https://github.com/mozilla-rally/rally-core-addon/pull/383): Adds IRB consent notice into the Core Add-On.
```

Let's not add blank lines between entries. I fixed this in another PR, you'll get a merge conflict once you rebase :)